bot/exts/info/reddit.py

```diff
@@ -179,7 +180,7 @@ async def get_top_posts(self, subreddit: Subreddit, time: str = "all", amount: i
         for post in posts:
             data = post["data"]
 
-            text = data["selftext"]
+            text = unescape(data["selftext"])
             if text:
```

Comment:
The line above could be moved into this if statement as a walrus operator
```suggestion
            if text := unescape(data["selftext"]):
```